.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,17 @@
+---
+name: User story
+about: Suggest a feature from the perspective of a specific user role
```

Comment:
```suggestion
about: Suggest a feature from the perspective of a specific user
```

Reply:
Maybe "specific type of user" instead? "Specific user" makes it sound like one specific user?